src/components/pdf-viewer/pdf-viewer.scss

```diff
@@ -348,6 +353,17 @@
     }
 }
 
+.page-number{
+    color: #83888d;
+    padding: 0px 15px;
+    font-family: Open Sans,Lucida Grande,Helvetica,sans-serif;
```

Comment:
Remove hard-coded font-family references. We will let the consuming web-app determine the font family of the UI.